src/MSALWrapper/TokenFetcher.cs

```diff
@@ -59,14 +59,14 @@ public record Result
             string prompt,
             TimeSpan timeout)
         {
-            var authFlows = AuthFlowFactory.Create(
+            var authFlows = new AuthFlows(client, tenant, AuthFlowFactory.Create(

```

Comment:
We're now passing `client` and `tenant` in two places here... Why doesn't the `AuthFlowFactory` return an `AuthFlows`?

Reply:
As long as that's something we're consciously doing now and it gets adjusted later, I'm OK with it in this PR.